src/main/java/org/broadinstitute/hellbender/utils/read/SAMRecordToGATKReadAdapter.java

```diff
@@ -276,7 +276,7 @@ public Cigar getCigar() {
      */
     @Override
     public int numCigarElements(){
-        return samRecord.getCigar() == null ? 0 : samRecord.getCigar().numCigarElements();
+        return samRecord.getCigar() == null ? 0 : samRecord.getCigarLength();
```

Comment:
No need for the ternery here.  samRecord.getCigarLength() already handles the null cigar case.